data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,268 @@ event "remnant: pantica void sprites"
+			choice
+				`	"So this is not the first expedition here?"`
+				`	"Anything unusual in the data?"`
+					goto astraldata
+				`	"How much longer does the equipment need?"`
+					goto astralfinish
+			label astralfirst
+			`	"No, this is the forty-third such expedition. Sagittarius A* was one of the first places our explorers sought to visit after we acquired jump drives, and we have been coming back regularly ever since. I know some of our reconnaissance pilots consider visiting here a pilgrimage of sorts. It is humbling to be here myself after reviewing so many data sets of past expeditions." She looks thoughtful, staring out the window at the the hub of the galaxy, as if her thoughts are jumping from idea to idea.`
+			choice
+				`	"Anything unusual in the data?"`
+				`	"How much longer does the equipment need?"`
+					goto astralfinish
+			label astraldata
+			`	"There are many oddities about this place. Every visit helps revise our physics models a little closer. But they also reveal problems." She frowns, her gestures taking on a curious tone. "For instance, you will note that your ramscoops are not picking up anything. According to standard astronomical theory, this system should be a veritable river of suitable material, and yet there is nothing. Not even the normal background trace amounts we find in deep space. We have visited other black holes, and they conform to the standard models in this respect. But not Sagittarius A*." She pauses, thinking about it for a moment.`
+			choice
+				`	"Is the equipment loaded?"`
+				`	"How much longer does the equipment need?"`
+			label astralfinish 
+			`	"The equipment will be finished shortly, Captain <first>. I just need to make a few observations and we will be able to depart. In the meantime, I would recommend against maintaining an active cloak or engaging any systems that consume fuel. For whatever reason, Sagittarius A* continues to be the one place that the stellar wind is utterly devoid of anything to collect as fuel."`
+			`	"I will be going outside for a few minutes," Dawn declares. "I shall just be on the hull."`
+			choice
+				`	"Let me know when you are done."`
+					goto astraldone
+				`	"Would you mind if I came too?"`
+			`	"Not at all, Captain. The view should be remarkable, and so long as we stay inside the shields, the radiation will not pose a problem." You quickly don your spacesuit and join Dawn outside on the hull. As promised, the view is indeed magnificent. Dawn signs occasionally, pointing out various features, and you appreciate that out here, sound would just be incongruous with the natural splendor.`
```

Comment:
They got some high-tech stuff if you can make out exact finger movements in spacesuits well enough for sign language to work. Just a random thought. Neat!